### PR TITLE
Language API changes (in progress) - DO NOT MERGE before Nov 15

### PR DESCRIPTION
**Pull request by puneith**

### CODE REVIEW

File: language/api/analyze.py [hunk 1]
```diff
@@ -24,15 +24,33 @@
...
+                 'version=v1&labels=GOOGLE_INTERNAL&key={}')
+
+# TODO UNCOMMENT - when discovery is public
+# def get_service():
+#     credentials = GoogleCredentials.get_application_default()
+#     scoped_credentials = credentials.create_scoped(
+#         ['https://www.googleapis.com/auth/cloud-platform'])
+#     http = httplib2.Http()
+#     scoped_credentials.authorize(http)
+#     return discovery.build('language', 'v1', http=http)
```
Comment (jerjou) [inline language/api/analyze.py]: FYI you can still do this. Just add the parameter `discoveryServiceUrl`:

```python
return discovery.build(
    'language', 'v1', http=http,
    discoveryServiceUrl='https://{api}.googleapis.com/$discovery/rest?version={apiVersion}&labels=GOOGLE_INTERNAL')

Comment (puneith) [inline language/api/analyze.py]: That's cool. Changes made.

**Comment by puneith** [comment]
@jerjou @jonparrott Please review.

File: language/api/analyze.py [hunk 2]
```diff
@@ -25,13 +25,20 @@
 from oauth2client.client import GoogleCredentials
 
 
+# TODO REMOVE - when discovery is public
```
Comment (jerjou) [inline language/api/analyze.py]: Can you remove this yet?

Comment (puneith) [inline language/api/analyze.py]: Not yet.

File: language/api/README.md [hunk 3]
```diff
@@ -38,7 +38,9 @@ where `<command>` is one of:  `entities`, `sentiment`, or `syntax`.
 
 The script will write to STDOUT the json returned from the API for the requested feature.
 
-For example, if you run:
```
Comment (jonparrott) [inline language/api/README.md]: I've recently updated all the READMEs to be autogenerated. Can you rebase?

File: language/api/analyze.py [hunk 4]
```diff
@@ -25,13 +25,20 @@
...
+
 def get_service():
     credentials = GoogleCredentials.get_application_default()
     scoped_credentials = credentials.create_scoped(
         ['https://www.googleapis.com/auth/cloud-platform'])
     http = httplib2.Http()
     scoped_credentials.authorize(http)
-    return discovery.build('language', 'v1beta1', http=http)
+    return discovery.build('language', 'v1',
+                           http=http,
```
Comment (jonparrott) [inline language/api/analyze.py]: Can you use `credentials=credentials` instead of `http=http`?

Comment (puneith) [inline language/api/analyze.py]: Not until discovery is public. I have added a TODO.

Comment (jerjou) [inline language/api/analyze.py]: Have you tried recently? I think there was a change to the api client recently (possibly made by Jon) that retried getting the discovery doc if it gets a permission-denied error.

Comment (jerjou) [inline language/api/analyze.py]: Oh well :-/

**Comment by jonparrott** [comment]
That bug is still open. :P

**Comment by puneith** [comment]
@jonparrott discovery public and pending changes made.

File: language/api/analyze.py (hunk 4 above)
Comment (jonparrott) [inline language/api/analyze.py]: break at opening `(` to avoid hanging indents. (I should make this into a t-shirt)

**Pull request merged**